specification/managedservices/resource-manager/Microsoft.ManagedServices/preview/2020-02-01-preview/managedservices.json

```diff
@@ -745,29 +745,29 @@
               "properties": {
                 "description": {
                   "type": "string",
-                  "description": "Description of the registration definition."
+                  "description": "The description of the registration definition."
                 },
                 "authorizations": {
                   "type": "array",
                   "items": {
                     "$ref": "#/definitions/Authorization"
                   },
-                  "description": "Authorization tuple containing principal id of the user/security group or service principal and id of the build-in role."
+                  "description": "The collection of principals in the managedBy tenant with permanent access to the managed tenant."
                 },
                 "eligibleAuthorizations": {
                   "type": "array",
                   "items": {
                     "$ref": "#/definitions/EligibleAuthorization"
                   },
-                  "description": "Eligible PIM authorization tuple containing principal id of the user/security group or service principal, id of the built-in role, and just-in-time access policy setting"
+                  "description": "The user ID, role, and just-in-time access policy for a principal in the managedBy tenant."
```

Comment:
>The user ID, role, and just-in-time access policy for a principal in the managedBy tenant." [](http://example.com/codeflow?start=34&length=91)

why eligible authorizations is having different approach of description than authorizations.